.env

```diff
@@ -3,11 +3,10 @@ RUST_LOG=info,sqlx::query=warn
 
 CDN_URL=https://cdn.modrinth.com
 
-DATABASE_URL=postgresql://labrinth@localhost/labrinth
+DATABASE_URL=postgresql://labrinth:labrinth@localhost/labrinth
```

Comment:
why?

Reply:
In the latest builds of postgres, removing the password for the user spits warnings in the logs, it's reflected in the docker-compose.yml file

Reply:
yeah, of course it does, it's set up to accept connections without authentication. that's intentional